simplified-main/src/main/res/values/stringsBoot.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
 <resources>
+  <string name="bootingAudioBookExtensions">Starting Audio Book extensions…</string>
```

Comment:
Instead of having the rich text ellipsis hard coded it's usually preferable to use the unicode entity.

```
<string name="bootingAudioBookExtensions">Starting Audio Book extension&#8230;</string>
```

Less fragile and it makes it easier to do things like grep for that specific entity.

Reply:
This one was an IDE suggestion - the original was three periods "...". I agree it's better to use the unicode entity and let the XML parser expand it.

Reply:
Sounds good. Should be an easy search and replace when you're working on the translation ticket.